package.json

```diff
@@ -8,13 +8,19 @@
   "license": "MIT",
   "type": "module",
   "devDependencies": {
+    "@vuepress/client": "2.0.0-beta.51",
     "remark-lint": "^9.1.1",
+    "vue": "^3.2.39",
     "vuepress": "2.0.0-beta.51"
   },
   "scripts": {
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
     "lint": "yarn remark docs/ --use remark-preset-lint-recommended -- use remark-preset-link-markdown-style-guide"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
Disclaimer: This is clearly a subjective thing and it may be to some extent stupid to manually edit a file, that's largely meant to be edited by the computer!

However, in those instances when a human being actually needs to review a package manifest, I find it helpful when `dependencies` and `devDependencies` are next to each other, just to prevent scrolling through a potentially long file.

Reply:
That wasn't me editing it either. That was pnpm....it added anything I told it to install.